src/plugin.js

```diff
@@ -15,6 +15,7 @@ const defaults = {
   timeout: 45 * 1000,
   dismiss: defaultDismiss,
   progressDisabled: false,
+  durationTolerance: 0.5,
```

Comment:
0.5 is an arbitrary low number. For reference here's an issue with a source that stops 0.6 near the end. Again this would only kick in if the player is starting to stall.